exporters/stats/signalfx/build.gradle

```diff
@@ -19,3 +20,10 @@ dependencies {
 
     signature "org.codehaus.mojo.signature:java17:+@signature"
 }
+
+shadowJar {
```

Comment:
Why do we need this?

Reply:
Because we don't want implementation and exporters to access opencensus-api's internal package. Instead we want to use a shaded dependency by the other artifacts, as suggested by @sebright in https://github.com/census-instrumentation/opencensus-java/pull/1114#pullrequestreview-109828652.

Reply:
If that is the case then it should be in common directory and implementations can use it.

Reply:
Sounds reasonable, I think we can make [common.TimeUtils](https://github.com/census-instrumentation/opencensus-java/blob/master/api/src/main/java/io/opencensus/common/TimeUtils.java) public and have this method be part of it.